lms/views/lti_launches.py

```diff
@@ -95,6 +95,18 @@ def launch_lti(request, lti_key=None, context_id=None, document_url=None,
     return _view_document(request, document_url=document_url, jwt=jwt)
 
 
+def _check_params(lti_params):
```

Comment:
Had to refactor because I got a pylint function too complex error.

Reply:
Ok, this looks good. In the future I think we might want to change this `_check_params` function into a global function for checking all necessary LTI launch params in one place, and have it be called by each view that needs the params. `lms/validation/lti_launch_params.py` or something.